dockers/docker-fpm-frr/frr/supervisord/supervisord.conf.j2

```diff
@@ -141,6 +141,20 @@ stderr_logfile=syslog
 dependent_startup=true
 dependent_startup_wait_for=bgpd:running
 
+{% if DEVICE_METADATA.localhost.frr_mgmt_framework_config is defined and DEVICE_METADATA.localhost.frr_mgmt_framework_config == "true" %}
+{% else %}
+[program:staticroutebfd]
+command=/usr/local/bin/staticroutebfd
+priority=6
```

Comment:
is the priority and dependent_startup_wait_for setup correctly here ? Just want to make sure we have thought of any race-condition that can occur

Reply:
Here try to use same priority and dependency with program bgpcfgd program, because the static route update need to go through bgpd eventually (staticroutebfd update redis_db APPL_DB STATIC_ROUTE_TABLE, and bgpcfgd handle the table update and update static route through bgpd).